### PR TITLE
Add new dashboard for program monitoring viz

### PR DESCRIPTION
**Pull request by rcrichton**
* includes the viz queries
* pipeline support for the new data element required

Requires the changes in this PR asa well: https://github.com/jembi/openhim-mediator-mpi-updater/pull/6

PLAT-496

### CODE REVIEW

File: packages/disi-logstash/pipeline/fhir-encounter.logstash.conf [hunk 1]
```diff
@@ -96,11 +107,17 @@ filter {
...
   }
 }
 
 output {
+  stdout {
+    codec => rubydebug {
+      metadata => true
+    }
+  }
+
```
Comment (nour-borgi) [inline packages/disi-logstash/pipeline/fhir-encounter.logstash.conf:120]: I think you forgot to remove this

File: packages/disi-logstash/pipeline/fhir-medication-dispense.logstash.conf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    whitelist_names => [ "hivCare", "@timestamp" ]
+  }
+}
+
+output {
+  stdout {
+    codec => rubydebug {
+      metadata => true
+    }
+  }
```
Comment (nour-borgi) [inline packages/disi-logstash/pipeline/fhir-medication-dispense.logstash.conf:45]: This one also

File: packages/disi-logstash/pipeline/fhir-encounter.logstash.conf (hunk 1 above)
Comment (rcrichton) [reply packages/disi-logstash/pipeline/fhir-encounter.logstash.conf:120]: Oops, good catch :)

**Review by tumbledwyer — APPROVED** [review]
LGTM

**Review by nour-borgi — APPROVED** [review]

**Pull request merged**